src/map_command_handler.hpp

```diff
@@ -190,10 +190,36 @@ class map_command_handler
 		}
 		else if (help_on_unknown_) {
 			utils::string_map symbols;
+			// Get the input command on a string
+			std::string string_user = get_cmd();
+			// Initialize the distance and the command proposal bool
+			int distance = 0;
+			bool has_command_proposal = false;
+			// Compare the input with every command (excluding alias)
+			for(typename command_map::value_type i : command_map_) {
```

Comment:
This should use a structured binding:
```cpp
for(const auto& [key, index] : command_map_)
```
(`i.first` being `key` and `i.second` being `index`)